test/integration/remove_stickers_from_cart_test.rb

```diff
@@ -0,0 +1,60 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.